templates/5_rename_and_reorganize.ipynb

```diff
@@ -5,104 +5,117 @@
    "id": "ed3314da-20ad-4047-a2bd-be6822b232f8",
```

Comment:
<p>This can be combined with cell above</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/222/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>